frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/select_option_cell/extension.dart

```diff
@@ -112,7 +113,11 @@ class SelectOptionTag extends StatelessWidget {
               name,
               fontSize: FontSizes.s11,
               overflow: TextOverflow.ellipsis,
-              color: AFThemeExtension.of(context).textColor,
+              color: ColorAnalyzer().getAppropriateTextColor(
+                color,
+                Theme.of(context).extension<AFThemeExtension>()!.textColor,
+                Theme.of(context).extension<AFThemeExtension>()!.textColorAlt,
```

Comment:
You're still not using `AFThemeExtension.of(context)`, it's my third time mentioning it..
```suggestion
                AFThemeExtension.of(context).textColor,
                AFThemeExtension.of(context).textColorAlt,
```

Reply:
Oh..so sorry for this. i must have left it in the final commit...